src/components/common/blocks/user-address-stats/index.js

```diff
@@ -32,7 +32,7 @@ class UserAddressStats extends React.Component {
     const stake = truncateNumber(lockedDgdStake || 0);
     const dgd = truncateNumber(lockedDgd || 0);
     return (
-      <Stats>
+      <Stats className="stats">
```

Comment:
Let's not use `className`s since we're already using styled components. You can just add the css as is in the `<Stats/>` styled component without adding a class.

Reply:
got it, changed, is this now ok?

Reply:
Yep, this is good for me. Thanks @Daivasmara! :tada: